shenyu-admin/src/test/java/org/apache/shenyu/admin/service/register/AbstractContextPathRegisterServiceTest.java

```diff
@@ -55,20 +58,56 @@ class AbstractContextPathRegisterServiceTest {
     private RuleServiceImpl ruleService;
 
     @Mock
-    private RegisterExecutionRepository registerExecutionRepository;
+    private LockRegistry registry;
 
     @Test
     public void testRegisterContextPath() {
         MetaDataRegisterDTO dto = MetaDataRegisterDTO.builder().build();
         dto.setContextPath("Context_Path");
         dto.setAddPrefixed(true);
-        RegisterExecutionLock registerExecutionLock = mock(RegisterExecutionLock.class);
         when(selectorService.registerDefault(dto, PluginEnum.CONTEXT_PATH.getName(), "")).thenReturn("Context_Path_Selector_Id");
-        when(registerExecutionRepository.getLock(any())).thenReturn(registerExecutionLock);
+        // org.springframework.integration.jdbc.lock.JdbcLockRegistry.JdbcLock is private and cannot be mocked directly so we mock the LockRegistry and return a mock MockLock
+        // here  mock ReentrantLock   cause cpu usage 100% in the jdk 19 20 21 environment
+        when(registry.obtain(any())).thenReturn(mock(MockLock.class));
```

Comment:
i change this mock ReentrantLock   cause cpu usage 100% in the jdk 19 20 21 environment